### ISSUE TITLE
Empty live region

### ISSUE THREAD

**Issue by JenM00re** [Swarthmore/accessible-docs#56]
> # [Alert: Empty Live Region](https://www.ssa.gov/accessibility/andi/help/alerts.html#live_region_empty).
> 
> ## Why did ANDI alert this?
> This element has been designated as a live region, but its contents are empty (no innerHTML).
> 
> ## Why is this an accessibility concern?
> Because screen readers only monitor a live region's contents for changes, a screen reader will not presently announce anything for this live region.
> 
> ## What should be done?
> This may not be an issue if the live region is supposed to be empty at a given moment. When the live region is dynamically updated with content changes, refresh ANDI and check the output. If the output still has this alert, it is likely that the live region is not being used properly. Screen readers do not monitor a live region's HTML attributes for changes. For example, if the live region's aria-label value is dynamically changing, a screen reader automatically speak the changes. Ensure that dynamic content changes to the live region are occurring within the innerHTML of the live region.
> 

![ANDI test showing empty live region error on a11ydocs login page](https://github.com/user-attachments/assets/bf092e37-f871-4328-bc0a-094c3b7206b1)

 - OS: Windows 10 64-bit
 - Browser Chrome
 - Version 135.0.7049.115 (Official Build) (64-bit)

**Comment by jerry-604** [comment]
https://nextjs.org/docs/architecture/accessibility
